tests/perfalert/test_auto_perf_sheriffing/test_secretary_tool.py

```diff
@@ -22,11 +22,18 @@ def get_middle_index(successful_jobs):
     return index_in_range
 
 
+def set_record_job_type(record, job_type_id=JOB_TYPE_ID):
+    record.job_type = JobType.objects.filter(id=job_type_id).first()
```

Comment:
You don't need to fetch the `JobType` instance. `job_type_id` is accessible as attribute of `record`.